src/tsconfig.json

```diff
@@ -113,6 +113,7 @@
         "rules/tripleEqualsRule.ts",
         "rules/typedefRule.ts",
         "rules/typedefWhitespaceRule.ts",
+        "rules/unboundFunctionThisRule.ts",
```

Comment:
seems like the wrong file name here. If you run grunt locally it should automatically fix this